src/home/Footer.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./Footer.css";
+import { Link } from "react-router-dom";
+import github from "./../assets/images/github.svg";
+import linkdin from "./../assets/images/LinkedIn.svg";
+import twitter from "./../assets/images/Twitter.svg";
+import facebook from "./../assets/images/facebook.svg";
+export default function Footer() {
+  return (
+    <section id="footer">
+      <div className="footer-container">
+        <div className="main-flexbox">
+          <div className="flexbox-item">
+            <img className="logo" src="#" alt="Logo"></img>
+            <h4 className="text">
+              We are known for connecting top talents with top comapnies. We
+              make great possibilities happen.
+            </h4>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Explore</h3>
+            <ul className="list">
+              <Link to="/" style={{ textDecoration: "none" }}>
+                <li className="li">Home</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">About Us</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">Blogs</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">FAQ</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">Contact Us</li>
+              </Link>
+            </ul>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Visit</h3>
```

Comment:
I think this can be removed as we don't have a visit location.

Reply:
Hi @Rahulm2310 @mtreacy002  , should i remove this section ?

Reply:
@naveen8801 , I believe @Rahulm2310 has answer your question so you can remove this 😉. @Vuyanzi and @isabelcosta unless you'd prefer this `Visit` to show AnitaB.Org mailing address as per anitab.org website? I believe it would be best to keep this consistent with other projects under AnitaB.Org Open Source.
<img width="777" alt="Screen Shot 2021-04-24 at 11 16 14 am" src="https://user-images.githubusercontent.com/29667122/115942664-ab6ab180-a4ee-11eb-92ef-b82bdf9d102b.png">

Reply:
@mtreacy002 is it possible to have AnitaB.orgs mailing address there?

Reply:
@Vuyanzi , I believe @isabelcosta has given the answer to this in her comment [below](https://github.com/anitab-org/bridge-in-tech-web/pull/228#issuecomment-826307730) 😉